unified-codes/apps/web/src/propertyFormats.ts

```diff
@@ -0,0 +1,35 @@
+const propertyFormats = [
```

Comment:
You did this just to hurt me, didn't you!?

Reply:
I thought you might take exception to this one.. sorry! I know it isn't great, but wanted to get something out.